src/components/InfiniteHits/InfiniteHits.js

```diff
@@ -64,13 +81,18 @@ InfiniteHits.propTypes = {
     emptyRoot: PropTypes.string.isRequired,
     list: PropTypes.string.isRequired,
     item: PropTypes.string.isRequired,
+    loadPrevious: PropTypes.string.isRequired,
+    disabledLoadPrevious: PropTypes.string.isRequired,
     loadMore: PropTypes.string.isRequired,
     disabledLoadMore: PropTypes.string.isRequired,
   }).isRequired,
   hits: PropTypes.array.isRequired,
   results: PropTypes.object.isRequired,
+  hasShowPrevious: PropTypes.bool.isRequired,
+  showPrevious: PropTypes.func,
```

Comment:
Both functions are required, we always provide both function.

Reply:
`showMore` was not marked as required so I did not maked `showPrevious` as required.

If I update `showPrevious`, do I also update `showMore` to mark it as required?

Reply:
Yep!